### ISSUE TITLE
igx-nav-drawer uses aside semantic element with incompatible role

### ISSUE THREAD

**Issue by kdinev** [IgniteUI/igniteui-angular#14847]
## Description  
The `aside` element is semantic for `role="complementary"` but is being applied `role="navigation"` in the `igx-nav-drawer`.

 * igniteui-angular version: 18.1.x
 * browser: all

## Steps to reproduce  

1. Define a nav drawer with template and mini template
2. Define some nav items
3. Run lighthouse analysis on the application

## Result  
There's an accessibility issue reported.

## Expected result  
There should be no accessibility issues reported.

## Attachments  
![image](https://github.com/user-attachments/assets/4068c191-fbbd-4c0c-8a1d-8653093f4f6d)
